cmd/exchange_template/types.tmpl

```diff
@@ -0,0 +1,7 @@
+{{define "types"}}
+package {{.Name}}
+{{end}}
+For efficiency, a JSON to Golang converter can be used: https://mholt.github.io/json-to-go/.
```

Comment:
ATM these notes don't have any use, these can instead by commented and with the {{end}} placed afterwards